### DISCUSSION TITLE
Trigger this poll through API?

### DISCUSSION THREAD

**Discussion by evanzummeren** [polppol/openpollslack-i18n#18] [General]
This is a great poll. Thank you for all the work and effort on this. Much appreciated.

Do you know if it's possible to trigger this through the chat_postMessage API? I was trying to just post ```/poll "Q" "A1" "A2" etc, but it just treats it as flat text.

**Comment by polppol** [comment]
Hi,
I don't think Slack can do that
as well as if you try to call some app using slack command in Workflow it also not working.

https://stackoverflow.com/questions/45350521/sending-command-via-slack-api
